widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java

```diff
@@ -203,7 +203,7 @@ private void navigateToChat() {
         Intent newIntent = new Intent(getApplicationContext(), ChatActivity.class);
         newIntent.putExtra(GliaWidgets.COMPANY_NAME, sdkConfiguration.getCompanyName());
         newIntent.putExtra(GliaWidgets.QUEUE_ID, sdkConfiguration.getQueueId());
-        newIntent.putExtra(GliaWidgets.CONTEXT_URL, sdkConfiguration.getContextUrl());
```

Comment:
May be we should still do `putExtra(GliaWidgets.CONTEXT_URL)` to keep compatibility for deprecated `CONTEXT_URL`? Otherwise, integrators who still use it, will loose their context. Or it's okay to drop it?

Reply:
It has no effect because for security purposes it removed completely from backend.